### PR TITLE
NR-155179 Azure qs dashboards

### PR DESCRIPTION
**Pull request by rajut-xrg**
# Summary
Added dashbaords related to azure, below are the list
1. Azure Logic Apps
2. Azure VPN Gateways
3. Azure SQL
4. Azure SQL managed instance
5. Azure Front Door
6. Azure Load Balancer
7. Azure Virtual machine scale sets
8. Azure SQL Managed Instances
9. Azure Redis Cache

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### CODE REVIEW

**Comment by aswanson-nr** [comment]
not stale

File: quickstarts/azure/azure-app-gateway/config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+slug: azure-app-gateway
+description: |-
+  ## What is Azure App Gateway?
+
+ Azure Application Gateway is a web traffic (OSI layer 7) load balancer that enables you to manage traffic to your web applications.
```
Comment (sarahkitten) [inline quickstarts/azure/azure-app-gateway/config.yml:5]:
```suggestion
  Azure Application Gateway is a web traffic (OSI layer 7) load balancer that enables you to manage traffic to your web applications. 
```

File: quickstarts/azure/azure-app-gateway/config.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - name: Azure App Gateway installation docs
+    description: |
+      Monitor Azure App Gateway by connecting Azure to New Relic.
+    url: https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-application-gateway-monitoring-integration/
+keywords:
+  - azure
+  - application gateway
+  - app gateway
+installPlans:
+  - azure-infrastructure-monitoring
```
Comment (pkudikyala) [inline quickstarts/azure/azure-app-gateway/config.yml:34]:
```suggestion
installPlans:
  - azure-infrastructure-monitoring
```

File: quickstarts/azure/azure-vpn-gateways/config.yml [hunk 3]
```diff
@@ -34,3 +34,5 @@ installPlans:
   - azure-infrastructure-monitoring
 dataSourceIds:
   - microsoft-azure
+dashboards:
+  - azure-vpn-gateway
```
Comment (pkudikyala) [inline quickstarts/azure/azure-vpn-gateways/config.yml:38]:
```suggestion
  - azure-vpn-gateways
```

**Review by pkudikyala — CHANGES_REQUESTED** [review]
Please update requested changes.

**Comment by d3caf** [comment]
Closing because this was done in #2264

**Pull request closed without merging**